core/src/main/java/com/linecorp/armeria/internal/server/FileAggregatedMultipart.java

```diff
@@ -58,13 +60,20 @@ public ListMultimap<String, MultipartFile> files() {
 
     public static CompletableFuture<FileAggregatedMultipart> aggregateMultipart(ServiceRequestContext ctx,
                                                                                 HttpRequest req) {
+        return aggregateMultipart(ctx, req, Collections.emptyList());
+    }
+
+    public static CompletableFuture<FileAggregatedMultipart> aggregateMultipart(
+            ServiceRequestContext ctx, HttpRequest req, List<String> parameters) {
         final Path destination = ctx.config().multipartUploadsLocation();
         return Multipart.from(req).collect(bodyPart -> {
             final String name = bodyPart.name();
             assert name != null;
+            if (!parameters.isEmpty() && !parameters.contains(name)) {
```

Comment:
I prefer to silently discard it because it might already process all the necessary files.
Let's say that there are 10 files and the last one is the one that the service doesnt' need.
This will reject the request after processing all the necessary files.

Reply:
Rather than rejecting it by throwing an exception (the loud way), I'll modify it to ignore unintended files (the quiet way)

📝 leaves memo for myself to remember later.